docs/outline.md

```
@@ -0,0 +1,125 @@
+{ "artifact_id": "/Users/me/proj/CLAUDE.md", "node_id": "1.2.3" }
+```
+
+Returns `{ success, node, text, line_count }`. Reads the source file again (cheap), checks
+mtime/size against the stored outline, and slices `lines[line_start..line_end]`.
+
+If the file has changed since build → `{ success: false, stale: true, error: "outline stale, rebuild needed" }`.
+The agent should call `memory_build_outline` again.
+
+## REST endpoints
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)